### PR TITLE
added unit tests for the javascript support in OfflineTs

### PR DESCRIPTION
**Pull request by Nandita-Khemnani**

### CODE REVIEW

File: test/js/JsObserverSpec.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+var chai = require("chai");
+var TestObserver, serviceStatus = require("../fixture/TestObserver");
```
Comment (sasidakh) [inline test/js/JsObserverSpec.js]: Is this file running??

File: spec/support/jasmine.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (SyedFayaz) [inline spec/support/jasmine.json]: Why this file is needed?

File: test/js/JsObserverSpec.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
+var chai = require('chai'),
+    expect = chai.expect,
+    TestObserver = require("../fixture/TestObserver").TestObserver,
+    PService = require("../fixture/TestObserver").PService,
+    serviceStatus = require("../fixture/TestObserver").serviceStatus,
+    TestObserverOne = require("../fixture/TestObserverOne").TestObserverOne,
+    StateType = require("../../dist/src/ServiceStatus").StateType,
+    SinonStub = require("sinon"),
```
Comment (SyedFayaz) [inline test/js/JsObserverSpec.js]: shouldn't this be require("sinon").SinonStub   ?

Comment (sasidakh) [reply test/js/JsObserverSpec.js]: this is not used i've removed it. and the other one.

**Pull request merged**